src/omnisharp/utils.ts

```diff
@@ -121,4 +134,58 @@ export async function debugTestStop(server: OmniSharpServer, request: protocol.V
 
 export async function isNetCoreProject(project: protocol.MSBuildProject) {
     return project.TargetFrameworks.find(tf => tf.ShortName.startsWith('netcoreapp') || tf.ShortName.startsWith('netstandard')) !== undefined;
+}
+
+function isBlazorWebAssemblyHosted(project: protocol.MSBuildProject): boolean {
+    if (!isBlazorWebAssemblyProject(project)) {
+        return false;
+    }
+
+    if (!project.IsExe) {
```

Comment:
Out of curiosity, why is this check important? I'd think the latter two checks would suffice.

Reply:
Just following suite with how O# typically checks "is runnable web" project.